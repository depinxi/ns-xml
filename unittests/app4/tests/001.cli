sub -c abc def ghi "This should be ok"