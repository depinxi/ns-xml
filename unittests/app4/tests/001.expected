CLI: "sub", "-c", "abc", "def", "ghi", "This should be ok"
Value count: 3
Values: "def", "ghi", "This should be ok"
Error count: 0
Subcommand: sub
switchA=False
argumentB=
multiargC=
sub_argumentD=abc
sub_switchE=False
